src_py/sprite.py

```diff
@@ -656,13 +685,13 @@ def add_internal(self, sprite, layer=None):
 
         if layer is None:
             try:
-                layer = sprite._layer
+                layer = sprite._layer  # noqa pylint: disable=protected-access, explicitly trying to break protected access
```

Comment:
`_layer` really probably should have been a public variable. Not really sure how to do that in a backwards compatible way. Maybe by making `_layer` a property alias to `layer`.

Reply:
or perhaps to a double underscore `__layer` ?

I'd vaguely like to turn all the public variables of the Sprite and DirtySprite class into properties, but perhaps that'd have some impacts on speed.